src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -1082,7 +1096,7 @@ static void _updateChildren(LottieGroup* parent, float frameNo, Inlist<RenderCon
                     break;
                 }
                 case LottieObject::Repeater: {
-                    _updateRepeater(parent, child, frameNo, contexts, ctx, exps);
+                    _updateRepeater(parent, child, frameNo, contexts, ctx, exps, groupLevel == level);
```

Comment:
Here, the level looks over-programmed. You might wanna just confirm whether the repeater is requested after the group.